certified-connectors/Revizto (Ireland)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1497 @@
+{
+  "basePath": "/",
```

Comment:
Swagger file is not in a correct format, kindly refer below links and modify duly. 

Template - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

Schema - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json